doc/sphinx/addendum/type-classes.rst

```diff
@@ -369,7 +369,7 @@ Summary of the commands
       The default value for instance locality will change in a future
       release. Instances added outside of sections without an explicit
       locality are now deprecated. We recommend using :attr:`export`
-      where possible.
+      where possible. This warning will raise an error by default.
```

Comment:
"This warning will raise an error by default." seems poorly worded--is it a warning or an error?  Perhaps you mean that the "warning" can be disabled through the standard mechanism and you can in include hyperlink to the command for doing so (which I don't remember offhand).  Perhaps you should refer to it as a "message" rather than a "warning"?

Reply:
Apparently warnings can be turned into errors as well as be disabled. This means that a user would be able to change its default behaviour back to be an error as well as disabling/enabling it.

Reply:
I'll change the wording to "This warning will error by default.".

Reply:
Not to be too picky, but "error" is not a verb in the dictionaries I looked in.